CMakeLists.txt

```diff
@@ -207,7 +207,7 @@ function(test_cxx CXX_RUNTIME_NAME IS_STDLIB)
 			"${CMAKE_BINARY_DIR}/CMake"
 			"${CMAKE_CURRENT_LIST_DIR}/CMake"
 			test_cxx_runtime
-			CMAKE_FLAGS "-DCXX_RUNTIME=${CXX_RUNTIME_LIB}")
+			CMAKE_FLAGS "-DCXX_RUNTIME=${CXX_RUNTIME_LIB}" "-DANDROID_ABI=${ANDROID_ABI}" "-DANDROID_NDK=${ANDROID_NDK}" "-DANDROID_PLATFORM=${ANDROID_PLATFORM}")
```

Comment:
Can we make this conditional on the target being android?

Reply:
Makes total sense, I’ve made that change and also added all other flags documented here:
https://developer.android.com/ndk/guides/cmake#variables